teleop_twist_joy:
  ros__parameters:
    # angular scale
    scale_angular: 1.0

    # linear scale
    scale_linear: 0.3

    # connection mode
    connection_mode: 1
